src/mudlet.h

```diff
@@ -414,6 +414,9 @@ class mudlet : public QMainWindow, public Ui::main_window
     bool muteGame() const { return mMuteGame; }
     bool mediaMuted() const { return mMuteAPI && mMuteGame; }
     bool mediaUnmuted() const { return !mMuteAPI && !mMuteGame; }
+    auto showSplitscreenTutorial() -> bool;
+    auto showedSplitscreenTutorial() -> void;
+    auto experiencedMudletPlayer() -> bool;
```

Comment:
Why change the format of the declaration? How does this make things easier to read? At first glance it looks like a change for change sake presumably as a result of a `(c++)++` rewrite of how things can be done...

:frowning:

Reply:
It is a change recommended by clang-tidy, and while it looked weird at first... it actually makes sense, for a few reasons!

1. most folks read left to right, and the function outputs after the function name makes more sense than function outputs before the function name
2. Python and Typescript (at minimum) also started this convention in recent years:
![image](https://github.com/user-attachments/assets/c5c54642-893a-4c0d-b9d7-c23d3f72732e)
<img width="841" alt="image" src="https://github.com/user-attachments/assets/b6ec9025-9a06-4b30-b038-a657683a0d27">
3. clang-tidy recommends this change by default, so I can expect to see this common trend in programming and in C++ to catch on: https://releases.llvm.org/17.0.1/tools/clang/tools/extra/docs/clang-tidy/checks/modernize/use-trailing-return-type.html

Let me know what you think. It seems strange at first, but it starts to make sense if you think about it, and if you program with multiple languages on a daily basis (I do, other contributors might as well - C++ isn't exactly the most popular language in the world, Typescript and Python are way more...) then it might help those folks. What do you think?

Reply:
Python depends on accurate indentation so it isn't a perfect language either though... :nauseated_face:

Reply:
OK, would you like to see this removed?

Reply:
Yeah.
> 1. most folks read left to right, and the function outputs after the function name makes more sense than function outputs before the function name

However C/C++ coders are used to seeing the type declaration before the identifier:
```cpp
int myNumber = 42; // Yep, this is the answer (to Life, the Universe and Everything)!
int myFunctionThatReturnsSomthing(); // Ah yes, it returns an "int"
```

Reply:
I merged without fixing this accidentally, but I'll change it back.